src/Components/Layout/StockLayout.jsx

```diff
@@ -5,6 +5,7 @@ import Cards from '../Cards/Cards'
 
 import "../Layout/StockLayout.css";
 import '../Cards/Cards.css'
+import { TableComponent } from "../table/TableComponent";
```

Comment:
First external imports occur that is imports from installed libraries, then 1 line space, then all internal imports that is files/components/functions that you have created, then 1 line space and then finally all css imports.